bundles/org.openhab.binding.linktap/README.md

```diff
@@ -0,0 +1,235 @@
+
+## Binding Configuration
+
+### Gateway Configuration
+
+| Name                  | Type   | Description                                                                                                                                                               | Recommended Values | Required | Advanced |
+|-----------------------|--------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|----------|----------|
+| host                  | String | The hostname / IP address of the gateway device                                                                                                                           |                    | Yes      | No       |
+| username              | String | The username if set for the gateway device                                                                                                                                |                    | No       | No       |
+| password              | String | The password if set for the gateway device                                                                                                                                |                    | No       | No       |
+| enableMDNS            | Switch | On connection whether the mDNS responder should be enabled on the gateway device                                                                                          | true               | No       | Yes      |
+| enforceProtocolLimits | Switch | If true data outside of the allowed ranges against the protocol will be logged and not sent                                                                               | true               | No       | Yes      |
+| enableJSONComms       | Switch | false by default for backwards compatibility, if using up to date firmware with no other local network applications set this to true, for more efficient communications   | true               | No       | Yes      |
+
+**NOTE** When enableMDNS is enabled, upon connection to the gateway option "Enable mDNS responder" is switched on 
+
+### Device Configuration
+
+| Name         | Type    | Description                                                           | Recommended Values | Required | Advanced |
+|--------------|---------|-----------------------------------------------------------------------|--------------------|----------|----------|
+| deviceId     | String  | The Device Id for the device under the gateway                        |                    | No (A,B) | No       |
+| deviceName   | String  | The name allocated to the device by the app. (Must be unique if used) |                    | No (B)   | No       |
+| enableAlerts | boolean | On connection whether the device should be configured to send alerts  | true               | No       | Yes      |
+
+**NOTE**
+
```

Comment:
```suggestion
```

Reply:
If I remove this it makes the document slight less readable? The First (A) is next to the Note, and the second (B) point is on a new line as currently. Are you okay if I leave this as is?

Reply:
This is what the linting suggests, its oke to leave it as is as it makes no difference to real rendering .